scripts/prohibit-direct-merges-to-master

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+current_branch=$(git branch --show-current)
+num_of_prs_from_current_to_master=$(gh pr list --state open --base master --head "$current_branch" | wc -l) # one line per one PR
```

Comment:
I'd only limit the execution of this script to the CI (based on the existence of `CI` variable)

Reply:
Or actually, now with a second look... maybe it makes sense to run this locally as well. 

But still, I'd avoid running this piece from the git `pre-commit` hook, see how `validate-markdown-links` is handled in `scripts/run-pre-build-checks`